pnpm-lock.yaml

```
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Apparently, later v8 versions write 6.0 again, after some people got really angry https://github.com/pnpm/pnpm/issues/6648.

```suggestion
lockfileVersion: '6.0'
```